### PR TITLE
Fixes a potential oversight in regards to Runechat layer placement.

### PR DESCRIPTION
**Pull request by CandleJaxx**
## About The Pull Request

edits the runechat layers to go over lighting/fullscreen fx while remaining under the hud, allowing for players to see people's messages while they're dying or blind, standardizes how runechat should be observed in accordance with #54522

## Why It's Good For The Game

incredibly funny to see bubbletext from cyborgs trying to heal you out of near-uncon crit and just reading screaming, also allows you to, when blind, 'hear' where people are talking from

## Changelog

:cl:

fix: edited runechat layers for clarity and consistency with old PRs

/:cl:

### CODE REVIEW

**Comment by MrMelbert** [comment]
image

This is pretty interesting but I doubt this can be considered a "fix"

**Comment by CandleJaxx** [comment]
> This is pretty interesting but I doubt this can be considered a "fix"

not sure- this is adding on to a previous fix, i suppose, but it's mostly just fleshing out the fix there. i labled it as such because it seemed like an oversight from that pr

**Comment by optimumtact** [comment]
it's either this or it's in the fov pr anyway so fuck it.

**Comment by LemonInTheDark** [comment]
@optimumtact this is something you want?

**Comment by optimumtact** [comment]
scratchy
... yes

File: code/__DEFINES/layers.dm [hunk 1]
```diff
@@ -136,10 +136,10 @@
 #define CAMERA_STATIC_PLANE 200
 
 ///Popup Chat Messages
-#define RUNECHAT_PLANE 250
+#define RUNECHAT_PLANE 501
 
 /// Plane for balloon text (text that fades up)
-#define BALLOON_CHAT_PLANE 251
+#define BALLOON_CHAT_PLANE 502
```
[the remark this answers is not in the document]
Comment (CandleJaxx) [reply code/__DEFINES/layers.dm]: set set! pologies, should be good now

File: code/__DEFINES/layers.dm [hunk 2]
```diff
@@ -135,14 +135,14 @@
...
+#define RUNECHAT_PLANE 501
 
 /// Plane for balloon text (text that fades up)
-#define BALLOON_CHAT_PLANE 251
-
-///Debug Atmos Overlays
-#define ATMOS_GROUP_PLANE 450
+#define BALLOON_CHAT_PLANE 502
 
 ///--------------- FULLSCREEN IMAGES ------------
```
Comment (LemonInTheDark) [inline code/__DEFINES/layers.dm:147]: Do you mind putting runechat and balloon above the fullscreen stuff? This sectioning is probably doing more harm then good, but I'd really like to maintain good sorting here

Comment (CandleJaxx) [reply code/__DEFINES/layers.dm:147]: ahh, alright- i think i got confused by what you meant earlier, one moment

Comment (CandleJaxx) [reply code/__DEFINES/layers.dm:147]: > above the fullscreen stuff?

and just to clarify- you mean like this, or am i misunderstanding?
image

Comment (CandleJaxx) [reply code/__DEFINES/layers.dm:147]: or, i assume now, 
image

Comment (LemonInTheDark) [reply code/__DEFINES/layers.dm]: All those layers are specific to the fullscreen plane, just move rune and balloon below CURSE_LAYER, could make a new section header if you'd like too


> ahh, alright- i think i got confused by what you meant earlier, one moment

Sall good, if you misunderstood s my fault for not explaining properly

Comment (CandleJaxx) [reply code/__DEFINES/layers.dm]: there we go, should be all set now!

**Review by LemonInTheDark — APPROVED** [review]
Chill

**Comment by cacogen** [comment]
you're not supposed to be able to see chat when you're in crit or unconcious though

**Comment by LemonInTheDark** [comment]
you "in theory" can only see what you can hear I'd think

**Comment by Ghommie** [comment]
> This is pretty interesting but I doubt this can be considered a "fix"

Looks good to me. The character is blind, but he can hear what's being said behind that transparent airlock - most probably his office - by who seem to be a pug (Mr Griff) and player character, so it's fair that the runechat text isn't hidden behind the blindness screen overlay, which I believe isn't intended behavior. So all in all, It just fixes some bugs that lead to some pejorated gameplay aspects. It's not a QoL from what I see, if that's what you're trying to say.

**Review by LemonInTheDark — APPROVED** [review]

File: code/__DEFINES/layers.dm [hunk 3]
```diff
@@ -153,6 +148,12 @@
 #define CRIT_LAYER 5
 #define CURSE_LAYER 6
 
+///--------------- FULLSCREEN RUNECHAT BUBBLES ------------
```
Comment (Mothblocks) [inline code/__DEFINES/layers.dm:151]: balloon text aint runechat i just fucked up the name

**Pull request merged**